batavia/core/builtins.js

```diff
@@ -24,158 +24,198 @@ batavia.builtins.<fn>.__doc__ = 'docstring from Python 3.4 goes here, for docume
 */
 
 batavia.builtins.__import__ = function(args, kwargs) {
+    // console.log("IMPORT", args[0], args[1], args[4]);
```

Comment:
This makes me think we should have some sort of "DEBUG" mode in batavia that we can turn on and off through init flags or args. 

This isn't me complaining about the comment, to be clear, this is me wishing I could conditionally enable the logging.